backend/src/utils/resourceUtils.ts

```diff
@@ -105,6 +105,21 @@ const fetchOrCreateDashboardCR = async (
     )
     .then((res) => {
       const dashboardCR = res?.body as DashboardConfig;
+      if (
+        dashboardCR &&
+        !dashboardCR.spec.dashboardConfig.disableKServe &&
+        !dashboardCR.spec.dashboardConfig.disableModelMesh
+      ) {
+        // return a merge between dashboardCR and blankDashboardCR but changing spec.disableKServe to true and spec.disableModelMesh to false
+        return _.merge({}, blankDashboardCR, dashboardCR, {
+          spec: {
+            dashboardConfig: {
+              disableKServe: true,
+              disableModelMesh: false,
```

Comment:
What if I set both `disableKServe` and `disableModelMesh` to `false` in the dashboard config? This will always disable the KServe although I want both of them to be enabled.

Reply:
Yes, good catch, let me update the PR